org.eclipse.winery.topologymodeler.ui/src/app/services/backend.service.ts

```diff
@@ -469,6 +479,15 @@ export class BackendService {
         return this.http.get(url + '/', { headers: this.headers });
     }
 
+    /**
+     * Requests all topology template ids
+     * @returns {Observable<string>}
+     */
+    requestAllTopologyTemplates(): Observable<any> {
```

Comment:
Why do you return `any` here? Don't you know the type you get?

Reply:
corrected